pkg/targets/reconciler/tektontarget/reaper.go

```diff
@@ -40,27 +40,27 @@ func reaperThread(ctx context.Context, r *reconciler) {
 
 	p, err := cloudevents.NewHTTP()
 	if err != nil {
-		log.Fatal("failed to create new HTTP based cloudevent: ", zap.Error(err))
+		log.Fatalw("Failed to create new HTTP based Cloudevent ", zap.Error(err))
```

Comment:
I would still remove the trailing spaces (not just this one, all of them).
Let's ship code in a clean state.